lectures/long_run_growth.md

```diff
@@ -294,10 +294,10 @@ plt.show()
 The preceding graph of percapita GDP strikingly reveals how the spread of the industrial revolution has over time gradually lifted the living standards of substantial
 groups of people  
 
-- Most of the growth happened in the past 150 years after the industrial revolution.
-- Percapita GDP's in the UK and the US, on the one hand, and in China, on the other, diverged from 1820 to 1940.
-- The gap has closed rapidly after 1950 and especially after the late 1970s.
-- These outcomes reflect complicated combinations of technological and economic-policy factors that students of economic growth try to understand and quantify
+- most of the growth happened in the past 150 years after the industrial revolution.
+- percapita GDP rised in the UK and the US, and in China, on the other hand, diverged from 1820 to 1940.
```

Comment:
"per capita GDP in the US and UK rose and diverged from that of China from 1820 to 1940"